mscore/pm.h

```diff
@@ -24,6 +24,7 @@
 #include "config.h"
 #include "mididriver.h"
 
+#include <QtGlobal>
```

Comment:
I didn't realize that all.h was included automatically.  Anyway, I've removed #include <QtGlobals.h> and have repushed.